### PR TITLE
 15.Jenkins

### PR DESCRIPTION
**Pull request by Korolev731**

### CODE REVIEW

File: IKorolev/15.Jenkins/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  gather_facts: no
+  become: yes
+  roles:
+    - jenkins
+    - nginx
+
+  tasks:
+    - name: Add nginx to jenkins group
+      shell: |
+        usermod -aG jenkins nginx
```
Comment (pluhin) [inline IKorolev/15.Jenkins/README.md:28]: Please use ansible module here

https://docs.ansible.com/ansible/2.9/modules/user_module.html

File: IKorolev/15.Jenkins/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    - name: Set httpd_can_network_connect
+      shell: |
+        setsebool -P httpd_can_network_connect 1
```
Comment (pluhin) [inline IKorolev/15.Jenkins/README.md:32]: Please use ansible module here
https://docs.ansible.com/ansible/2.9/modules/seboolean_module.html

File: IKorolev/15.Jenkins/play.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  gather_facts: no
+  become: yes
+  roles:
+    - jenkins
+    - nginx
+
+  tasks:
+    - name: Add nginx to jenkins group
+      shell: |
+        usermod -aG jenkins nginx
```
Comment (pluhin) [inline IKorolev/15.Jenkins/play.yaml:12]: Use ansible module

File: IKorolev/15.Jenkins/play.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+    - name: Set httpd_can_network_connect
+      shell: |
+        setsebool -P httpd_can_network_connect 1
```
Comment (pluhin) [inline IKorolev/15.Jenkins/play.yaml:16]: Use ansible module

**Review by pluhin — CHANGES_REQUESTED** [review]

File: IKorolev/15.Jenkins/play.yaml (hunk 4 above)
Comment (Korolev731) [reply IKorolev/15.Jenkins/play.yaml]: corrected

File: IKorolev/15.Jenkins/play.yaml (hunk 3 above)
Comment (Korolev731) [reply IKorolev/15.Jenkins/play.yaml]: corrected

File: IKorolev/15.Jenkins/README.md (hunk 2 above)
Comment (Korolev731) [reply IKorolev/15.Jenkins/README.md]: corrected

File: IKorolev/15.Jenkins/README.md (hunk 1 above)
Comment (Korolev731) [reply IKorolev/15.Jenkins/README.md]: corrected

**Review by pluhin — APPROVED** [review]

**Pull request merged**